src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -293,11 +293,20 @@ public void SendFunctionLoadRequests(ManagedDependencyOptions managedDependencyO
                         .Timeout(_functionLoadTimeout)
                         .Take(_functions.Count())
                         .Subscribe((msg) => LoadResponse(msg.Message.FunctionLoadResponse), HandleWorkerFunctionLoadError));
+
+                    _functionLoadTimeout = functionTimeout.Value > _functionLoadTimeout ? functionTimeout.Value : _functionLoadTimeout;
```

Comment:
I don't see a difference between this `_functionLoadTimeout` and the one defined on line 291, we can probably delete this line as `_functionLoadTimeout` is already defined. Or is there a reason we need to define it twice?

Reply:
Removed